tests/baselines/reference/library-reference-5.errors.txt

```diff
@@ -15,16 +12,13 @@
 ==== /node_modules/foo/node_modules/alpha/index.d.ts (0 errors) ====
     declare var alpha: any;
     
-==== /node_modules/bar/index.d.ts (1 errors) ====
+==== /node_modules/bar/index.d.ts (0 errors) ====
     /// <reference types="alpha" />
-                          ~~~~~
-!!! error TS4090: Conflicting definitions for 'alpha' found at '/node_modules/bar/node_modules/alpha/index.d.ts' and '/node_modules/foo/node_modules/alpha/index.d.ts'. Consider installing a specific version of this library to resolve the conflict.
-!!! error TS4090:   The file is in the program because:
-!!! error TS4090:     Type library referenced via 'alpha' from file '/node_modules/foo/index.d.ts'
-!!! error TS4090:     Type library referenced via 'alpha' from file '/node_modules/bar/index.d.ts'
-!!! related TS1404 /node_modules/foo/index.d.ts:1:23: File is included via type library reference here.
     declare var bar: any;
     
-==== /node_modules/bar/node_modules/alpha/index.d.ts (0 errors) ====
+==== /node_modules/bar/node_modules/alpha/index.d.ts (1 errors) ====
     declare var alpha: {};
+                ~~~~~
+!!! error TS2403: Subsequent variable declarations must have the same type.  Variable 'alpha' must be of type 'any', but here has type '{}'.
+!!! related TS6203 /node_modules/foo/node_modules/alpha/index.d.ts:1:13: 'alpha' was also declared here.

```

Comment:
I was going to say "the old message seemed useful because it said where the dupe was", but I guess this related line would be enough. Though it's moderately a shame to lose a message that gives a potential solution.

Reply:
The things work much better where you could have two typings that dont add to global and used by two packages and hence two different versions. This handles that well. We have this issue with module resolution. The only message we gave was with typeRef and it wasnt correct checking either so i feel better removing this custom logic for typeRef only and allows packages to use typeRef of different versions if they dont add conflicting global info.